client/src/components/TourGuide/steps.tsx

```diff
@@ -0,0 +1,54 @@
+import { Box, Divider, Typography } from '@material-ui/core';
+import { ReactourStep } from 'reactour';
+import { stepStyles } from './stepStyles';
+
+interface CardStepProps {
+  heading: string;
+  description: string;
+}
+
+const CardStep = ({ heading, description }: CardStepProps): JSX.Element => {
+  const classes = stepStyles();
+
+  return (
+    <Box>
+      <Typography variant="h4" color="primary" gutterBottom>
+        {heading}
+      </Typography>
+      <Divider light className={classes.divider} />
+      <Typography variant="body1">{description}</Typography>
+    </Box>
+  );
+};
+
+export const steps: ReactourStep[] = [
```

Comment:
There is no step for adding a column, I will add that in as it's a pretty hidden feature and probably best to highlight it.